README.adoc

```diff
@@ -284,7 +284,7 @@ supplied serial number matches the constraint, [hotspot=isValid file=2]`isValid(
 // ==================================================================================
 == Programmatically validating constraints
 
-Finally, create a service that can be used to programmatically validate the constraints
+Next, create a service that can be used to programmatically validate the constraints
```

Comment:
suggested edit:

Next, create a service to programmatically validate the constraints